lib/datadog/open_feature/binding/evaluator.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative 'resolution_details'
+
+module Datadog
+  module OpenFeature
+    module Binding
+      class Evaluator
+        def initialize(ufc_json)
```

Comment:
just my opinion: I think we should avoid abbreviations when we can, for people not familiar with OpenFeature language this doesn't mean anything